src/sentry/static/sentry/app/views/issueList/searchBar.tsx

```diff
@@ -102,9 +102,9 @@ class IssueListSearchBar extends React.Component<Props, State> {
    * @returns array of tag values that substring match `query`
    */
   getTagValues = async (tag: Tag, query: string): Promise<string[]> => {
-    const {tagValueLoader, projectIds} = this.props;
+    const {tagValueLoader} = this.props;
 
-    const values = await tagValueLoader(tag.key, query, projectIds);
```

Comment:
Was this intentional? I think we need the projectids to scope tag values to the right projects.

Reply:
If you look at the `tagValueLoader` function in `overview.tsx`, it doesn't use `projectIds`